### ISSUE TITLE
508-defect-1: Table within the benefit summary and service verification letter accordion is cut off on smaller screen sizes

### ISSUE THREAD

**Issue by sortizsh** [department-of-veterans-affairs/va.gov-team#45946]
## Point of contact

Josh Kim

## List of issues by severity
There are 3 tickets related to this issue: 
Main ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45650
Heading: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45944
Labelling: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45945
Responsive: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45946
Download: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45998
Visual Instructions: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45999
I've grouped these issues together as they're closely related and dependent on one another. Whatever solution we consider will likely need to take into account all of these issues together from a holistic perspective @skylerschain 

### Table cuts off on smaller screen sizes (Critical)
[This table doesn't follow design system guidance on responsive list tables.](https://design.va.gov/components/table); content that runs beyond the width of the device is inaccessible to mobile keyboard users.
<img width="518" alt="Screen Shot 2022-08-19 at 3 56 21 PM" src="https://user-images.githubusercontent.com/14154792/185697527-8a885a2e-be1d-451f-b590-889493be6d6e.png">

### Buttons and tables cut off on mobile view. Alert is too narrow to read. (Moderate)
<img width="452" alt="responsivebug" src="https://user-images.githubusercontent.com/14154792/184414632-f53be4ff-7f27-4668-9197-2db6fe0a7fff.png">
<img width="407" alt="Screen Shot 2022-08-12 at 12 43 09 PM" src="https://user-images.githubusercontent.com/14154792/184414635-19a894e9-4770-40f7-9b73-872e46a6d973.png">

### Download button needs review (Moderate)
The user is both generating the pdf and being navigated to a new page. We should clarify this in the language and styling of the component.

https://user-images.githubusercontent.com/14154792/184921694-3315ebc4-e781-447b-a942-357839887299.MP4

### Instructions rely on visual symbols (Moderate)
The first paragraph instructs users to expand boxes with "+"- a visual symbol that is not announced to screen reader users in the accordions. An immediate bandaid fix may be to align the aural experience with the instructions e.g., "To see an explanation of each letter, expand the _buttons_ below." However, this is likely a workaround to make up for an suboptimal design pattern.

https://user-images.githubusercontent.com/14154792/184918282-e7d465e0-0081-4fda-baed-65d5eaefcd53.MP4

## Reproduction steps

1. [Log in as user 299 and get to step 2](https://staging.va.gov/records/download-va-letters/letters/letter-list)
2. Open the benefit summary accordion

## Proposed solution or next steps

TBD, this requires further exploration with design, engineering, and content. My immediate thoughts are:
- we should look into past research on why the designers choose the accordion and table pattern (for example, how complex can these tables be? e.g. for military service history); if no strong design reasons exist, we may want to consider redesigning this
- we should consider using plain checkboxes (instead of checkboxes within a table)
- depending on the length of this, it may deserve to live on a separate page 

## Type of issue

- [x] Screenreader
- [x] Keyboard
- [ ] Focus
- [x] Headings
- [ ] Color
- [x] Zoom
- [x] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by joshkimux** [comment]
[Closing and re-opening more specific ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/50581)
